dali/sasha/saxref.cpp

```diff
@@ -1180,6 +1180,8 @@ class CNewXRefManager: public CNewXRefManagerBase
         cDirDesc *p = findDirectory(pdir.str());
         if (!p)
             return NULL;
+        if (isPlaneStriped&&p==root&&tail[0]=='d'&&readDigits(tail+1)!=0)
```

Comment:
Looks correct, but future readers, it could do with a comment describing why/what it is doing here.